etc/profile-a-l/imv.profile

```diff
@@ -0,0 +1,40 @@
+# Persistent local customizations
+include imv.local
+# Persistent global definitions
+include globals.local
+
+include allow-bin-sh.inc
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+
+include whitelist-runuser-common.inc
+
+apparmor
+caps.drop all
+net none
+nodvd
+noinput
+nonewprivs
+nosound
+notv
+nou2f
+novideo
+protocol unix
```

Comment:
Does it need `unix`?

Reply:
It seems that unix socket is required for wayland.

Reply:
> Can net none alone disable unix sockets?

No it is not a syscall related option, it is a namespace related option. To block all protocols, use `seccomp socket`. Anyway if it uses wayland ...